CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -0,0 +1,91 @@
+  - Document Pods using FQDN based policies must respect DNS TTL. ([#6230](https://github.com/antrea-io/antrea/pull/6230), [@tnqn])
+  - Document the limitations of Audit Logging for policy rules. ([#6225](https://github.com/antrea-io/antrea/pull/6225), [@antoninbas])
+  - Add the user document for Antrea native secondary network support. ([#6015](https://github.com/antrea-io/antrea/pull/6015) [#6042](https://github.com/antrea-io/antrea/pull/6042), [@jianjuns] [@antoninbas])
+- Increase the minimum supported Kubernetes version to v1.19. ([#6089](https://github.com/antrea-io/antrea/pull/6089), [@hjiajing])
+- Upgrade IPPool CRD to v1beta1 and make the subnet definition consistent with the one in ExternalIPPool CRD. ([#6036](https://github.com/antrea-io/antrea/pull/6036), [@mengdie-song])
+- Request basic memory for antrea-controller to adjust its OOM score to avoid being killed too early by K8s scheduler. ([#6233](https://github.com/antrea-io/antrea/pull/6233), [@tnqn])
+- Increase default rate limit of antrea-controller to improve performance for batch requests. ([#6231](https://github.com/antrea-io/antrea/pull/6231), [@tnqn])
+- Remove Docker Support for Antrea Agent on Windows. ([#6019](https://github.com/antrea-io/antrea/pull/6019), [@XinShuYang])
+- Publish the new split antrea-agent and antrea-controller images only by removing the unified image. ([#6182](https://github.com/antrea-io/antrea/pull/6182), [@antoninbas])
+- Avoid unnecessary DNS queries for FQDN rule of NetworkPolicy in Antrea Agent. ([#6200](https://github.com/antrea-io/antrea/pull/6200), [@tnqn])
+- Optimize package organization to reduce antctl binary size. ([#6037](https://github.com/antrea-io/antrea/pull/6037), [@tnqn])
+- Disable cgo for all Antrea binaries. ([#5988](https://github.com/antrea-io/antrea/pull/5988), [@antoninbas])
+- Stop using `projects.registry.vmware.com` for user-facing images. ([#6073](https://github.com/antrea-io/antrea/pull/6073), [@antoninbas])
+- Fix nil pointer dereference when ClusterGroup/Group is used in NetworkPolicy controller. ([#6077](https://github.com/antrea-io/antrea/pull/6077), [@tnqn])
+- Disable `libcapng` to make logrotate run as root in UBI images to fix an OVS crash issue. ([#6052](https://github.com/antrea-io/antrea/pull/6052), [@xliuxu])
+- Fall back to lenient decoding when strict decoding config fails to tolerate unknown fields and duplicate fields, which helps to improve antrea robustness. ([#6156](https://github.com/antrea-io/antrea/pull/6156), [@tnqn])
+- Reduce antrea cni binary size by removing unnecessary import package. ([#6038](https://github.com/antrea-io/antrea/pull/6038), [@tnqn])
+- Strip all debug symbols from Go binaries by default. ([#6035](https://github.com/antrea-io/antrea/pull/6035), [@antoninbas])
+- Skip loading openvswitch kernel module if it's already built-in. ([#5979](https://github.com/antrea-io/antrea/pull/5979), [@antoninbas])
```

Comment:
```suggestion
- Skip loading Open vSwitch kernel module if it's already built-in. ([#5979](https://github.com/antrea-io/antrea/pull/5979), [@antoninbas])
```

Reply:
This is kept as openvswitch in purpose because it's the module name, maybe I can add it as `openvswitch`.